  --- scripts/nanopolish_merge.py     2016-08-12 18:35:29.000000000 +0300
  +++ scripts/nanopolish_merge.py 2016-08-12 18:37:54.000000000 +0300
  @@ -1,3 +1,4 @@
  +#!/usr/bin/env python
   import sys
   import glob
   from Bio import SeqIO

  --- scripts/nanopolish_makerange.py 2016-08-12 18:35:44.000000000 +0300
  +++ scripts/nanopolish_makerange.py     2016-08-12 18:37:37.000000000 +0300
  @@ -1,3 +1,4 @@
  +#!/usr/bin/env python
   import sys
   from Bio import SeqIO

--- Makefile.bak        2016-08-12 18:29:05.000000000 +0300
+++ Makefile    2016-08-12 18:29:31.000000000 +0300
@@ -14,7 +14,7 @@
 CFLAGS=-O3
 CXX=g++
 CC=gcc
-HDF5=install
+#HDF5=install

 # Check operating system, OSX doesn't have -lrt
 UNAME_S := $(shell uname -s)
